<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsWindow</class>
 <widget class="QWidget" name="SettingsWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>360</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../resources.qrc">
    <normaloff>:/resources/icon@3x.png</normaloff>:/resources/icon@3x.png</iconset>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <widget class="QSpinBox" name="INPUT_Opacity">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>90</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="INPUT_Padding">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>90</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>180</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>English</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Polish</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="BTN_Save">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>260</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QComboBox" name="SELECT_Position">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>180</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Top Left</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Top Right</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Bottom Left</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Bottom Right</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>155</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Overlay Position</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>66</y>
     <width>181</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Overlay Padding</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>65</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Overlay Opacity</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>155</y>
     <width>241</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Language (not implemented yet)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>310</y>
     <width>641</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#aa0000;&quot;&gt;App needs to be restarted for changes to apply!&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="textFormat">
    <enum>Qt::TextFormat::RichText</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>270</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Install Launch Hook</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../../resources.qrc"/>
 </resources>
 <connections/>
</ui>
